lib/services/ios-debug-service.ts

```diff
@@ -93,6 +93,15 @@ class IOSDebugService extends DebugServiceBase implements IPlatformDebugService
 		}
 	}
 
+	protected getChromeDebugUrl(debugOptions: IDebugOptions, port: number): string {
+		const debugOpts = _.cloneDeep(debugOptions);
+		debugOpts.useBundledDevTools = debugOpts.useBundledDevTools === undefined ? false : debugOpts.useBundledDevTools;
+		debugOpts.useHttpUrl = debugOpts.useHttpUrl === undefined ? false : debugOpts.useHttpUrl;
```

Comment:
The idea is that we may have many different cases for the debug options in both platforms. That's why I prefer using two separate methods. Also I really want to assure that when ios-debug-service is used, we are using correct options, so testing only the base class is not an option.
I'll move this line to the `super` method, but I'll keep using the same logic.